eng/testing/linker/project.csproj.template

```diff
@@ -8,4 +8,23 @@
     <TargetingPackDir>{TargetingPackDir}</TargetingPackDir>
   </PropertyGroup>
 
+  <!-- Logic to override the default IlLink tasks that come from the SDK and use the one
+  we use in dotnet/runtime repo -->
```

Comment:
Nit: This comment is a little confusing.  "Use the one we use in dotnet/runtime repo"... for src?

Reply:
Sorry you are right that this is not very clear. This project template is the one used for generating the test projects for our trimming tests. We publish and run them out of proc, meaning that by default, they will use the ILLinking infrastructure that comes with the SDK. With the next few lines, what we are doing is that we are forcing these test projects to use the same ILLinking infrastructure that we use in the rest of the runtime repo even when they run out of proc.

And to answer your question, yes, for building runtime repo src projects we use a special version of the linker so that is the one we are switching to here.